content/news/2024_10_11_refreshing_grass_psc.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Results of GRASS GIS PSC elections"
+date: 2024-10-11T11:13:00+02:00
+layout: "news"
+author: GRASS Development Team
+---
+
+### Welcome to the Newly Elected Members of the GRASS GIS PSC!
+
+In mid-September, the GRASS GIS community began the election process 
+to fill the seats previously held by **Helena Mitasova**, **Helmut Kudrnovsky**, 
+**Martin Landa**, and **Michael Barton**. After nominations from the community, 
+*10 GRASS GIS contributors* stood as candidates for these four PSC positions.
```

Comment:
While specific contributions and community involvement is needed to vote, we purposefully have no such requirements for PSC candidates to allow any member of the community to be part of PSC. Let's use community members here not to create false impression about the general requirements.

```suggestion
*10 GRASS GIS community members* stood as candidates for these four PSC positions.
```